tasks/README.md

```diff
@@ -227,4 +227,6 @@ Name | Summary | Category
 `task273_europarl_classification.json`| Given bulgarian sentence and corresponding english translation, verify that the translation is right or wrong | Classification
 `task274_overruling_classification`	| Given a sentence, classify it into overruling or non-overruling| Classification
 `task275_enhanced_wsc_paraphrase_generation` | Given a sentence and an aspect, paraphrase the sentence changing that aspect | Text Modification
-`task276_enhanced_wsc_classification` | Given a sentence and its paraphrase, decide what is the difference between them. | Classification
\ No newline at end of file
+`task276_enhanced_wsc_classification` | Given a sentence and its paraphrase, decide what is the difference between them. | Classification
+`task288_gigaword_sentence_generation.json` | Given a text of article, generate headline (title) for the article | Sentence Generation
+`task289_gigaword_classification.json` | Given a text of article and corresponding title, generate yes if title is appropriate, otherwise generate no | Classification
```

Comment:
A simpler description would be:
Given the text of an article and its title, decide whether the title is appropriate for the article.

Reply:
Thanks. Updated.